server/src/main/java/io/deephaven/server/hierarchicaltable/HierarchicalTableViewSubscription.java

```diff
@@ -272,14 +276,14 @@ private void process() {
                 }
             }
             if (sendError) {
-                GrpcUtil.safelyError(listener, GrpcUtil.securelyWrapError(log, upstreamFailure, Code.DATA_LOSS));
+                GrpcUtil.safelyError(listener, errorTransformer.transform(upstreamFailure));
```

Comment:
I'm not sure if this context was important, but we are losing `Code.DATA_LOSS`.

Reply:
Yeah, that hurt me, too... -- But, it highlights the inability to differentiate different SRE status types when transforming non-SRE exceptions.